equinix/resource_network_device.go

```diff
@@ -442,6 +444,12 @@ func createNetworkDeviceSchema() map[string]*schema.Schema {
 			ConflictsWith: []string{neDeviceSchemaNames["Secondary"]},
 			Description:   neDeviceDescriptions["DiverseFromDeviceUUID"],
 		},
+		neDeviceSchemaNames["Tier"]: {
+			Type:        schema.TypeInt,
+			ForceNew:    true,
+			Optional:    true,
```

Comment:
Does the API return tier attribute all the time or just for C8000V device? If the API returns it, we need to add computed attribute here.

Reply:
it returns only for C8000V